### ISSUE TITLE
lat/lon metadata display inconsistent on iphone

### ISSUE THREAD

**Issue by cori** [laurent22/joplin#104]
### Operating system

- [ ] Windows
- [ ] macOS
- [ ] Linux
- [ ] Android
- [ X ] iOS

### Application

- [ ] Desktop
- [ X ] Mobile
- [ ] Terminal

When I create a note on the iPhone and show metadata, the lat/lon are empty. If I show on map it seems to find me just find, though, and when I've synced and opened the note on my Mac and then re-synced the metadata shows up.

**Before sync/re-sync**
![img_1cfd7b8dc164-1](https://user-images.githubusercontent.com/46317/34071064-e70a1c14-e235-11e7-9257-8b7b55e939e4.jpeg)

**After sync/re-sync**
![img_b54cd17f81fa-1](https://user-images.githubusercontent.com/46317/34071104-ccd009b6-e236-11e7-9070-1e94cdb02c3f.jpeg)

---

**Comment by cori** [comment]
Now that I can see the metadata in the terminal one thing that may be of use here is that the notes I created on my iPhone (all of which had the empty lay/lon problem when initially viewing the metadata) have different lat/lon than the ones I created in terminal or desktop, and all have the same lat/lon. That seems to indicate that the iPhone *Is* assigning a lat/lon (as opposed to it being assigned in the terminal or desktop and resynced and that the problem is in display. In addition they all have an altitude (which I assume comes from the iPhone’s gps), whereas the others do not. 

Not sure if this is helpful or not.
